src/compas_rhino/geometry/brep/brep.py

```diff
@@ -302,6 +302,39 @@ def from_cylinder(cls, cylinder):
         rhino_cylinder = cylinder_to_rhino(cylinder)
         return cls.from_native(rhino_cylinder.ToBrep(True, True))
 
+    @classmethod
+    def from_curves(cls, curves):
+        """Create a RhinoBreps from a list of planar face boundary curves.
+
+        Parameters
+        ----------
+        curves : : list of :class:`~compas.geometry.Curve` or :class:`~compas.geometry.Polyline`
+            The planar curves that make up the face borders of brep faces.
+
+        Returns
+        -------
+        list of :class:`~compas_rhino.geometry.RhinoBrep`
+
+        """
+        if not isinstance(curves, list):
+            curves = [curves]
+        faces = []
+        for curve in curves:
+            if isinstance(curve, Polyline):
+                rhino_curve = polyline_to_rhino_curve(curve)
+            else:
+                rhino_curve = curve_to_rhino(curve)
+            face = Rhino.Geometry.Brep.CreatePlanarBreps(rhino_curve)
+            if face is None:
+                raise BrepError("Failed to create face from curve: {} ".format(curve))
+            if isinstance(face, list) and len(face) > 1:
```

Comment:
if `face` is not `None` it's safe to assume `face` is a list so this check is superfluous.